public/data/food/processes.json

```diff
@@ -7550,7 +7516,7 @@
   {
     "categories": ["ingredient"],
     "comment": "",
-    "displayName": "Blé tendre Hors UE Conv.",
+    "displayName": "Blé tendre UE Conv.",
```

Comment:
the id is `soft-wheat-non-eu` and we're changing the displayName from `Hors UE` to `UE` but the LCI stays `Soft wheat GLO`. Is that normal ? 

(Now the displayName is incoherent with the id, we really need to go to uuid)

Reply:
Thanks! Julia has found the error and will fix it